war/src/main/js/api/plugins.js

```diff
@@ -10,12 +10,15 @@
 exports.recommendedPlugins = [
     "antisamy-markup-formatter",
     "credentials",
+    "github-branch-source",
     "junit",
     "mailer",
     "matrix-auth",
     "script-security",
     "subversion",
-    "translation"
+    "translation",
+    "workflow-aggregator",
```

Comment:
@jglick The rest of the list is whatever is currently bundled with Jenkins. Dependencies are getting pulled in automatically.